frontend/pages/hosts/details/cards/Software/SelfService/CategoriesMenu/CategoriesMenu.tsx

```diff
@@ -4,14 +4,14 @@ import classNames from "classnames";
 import LinkWithContext from "components/LinkWithContext";
 import TooltipTruncatedText from "components/TooltipTruncatedText";
 
-import { parseHostSoftwareQueryParams } from "../../HostSoftware";
+import { parseSelfServiceQueryParams } from "../SelfService";
 import { ICategory } from "../helpers";
 
 const baseClass = "categories-menu";
 
 export interface ICategoriesMenu {
   categories: ICategory[];
-  queryParams: ReturnType<typeof parseHostSoftwareQueryParams>;
+  queryParams: ReturnType<typeof parseSelfServiceQueryParams>;
```

Comment:
We're still passing the correct available-for-install/self-service query params to the API endpoint itself after this change right? Noticed they were dropped out of the test and want to make 100% sure we don't cause a regression here. Guessing it's fine because we're just talking about the address bar here but wanted to triple-check.

Reply:
Confirmed on main and this branch that:

API call remains:
`https://localhost:8080/api/latest/fleet/device/dc743278-f81c-4874-a4fc-8c9c42d5db2d/software?page=0&per_page=9999&order_key=name&order_direction=asc&self_service=true`

being populated by:
```
// These default params are not subject to change by the user
const DEFAULT_SELF_SERVICE_QUERY_PARAMS = {
  per_page: 9999, // Note: There is no API pagination on this page because of time constraints (e.g. categories and install statuses are not filtered by API)
  order_key: "name",
  order_direction: "asc",
  self_service: true,
  category_id: undefined,
} as const;
```